client/shade/pom.xml

```diff
@@ -279,6 +279,13 @@
                                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                                     <manifestEntries>
                                         <Multi-Release>true</Multi-Release>
+                                        <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
+                                        <Bundle-SymbolicName>org.jboss.client</Bundle-SymbolicName>
+                                        <Bundle-Version>1.0</Bundle-Version>
+                                        <Bundle-Name>JBoss EAP client library</Bundle-Name>
```

Comment:
Well, this is WildFly and not EAP, so JBoss EAP I guess is not suitable here. I suggested in a previous comment to look for a name from client/shade/pom.xml#name, although without "WildFly: ".

What about "Jakarta Enterprise Beans and Jakarta Messaging client library"? Would you be fine with using it?

Reply:
Hi @yersan,
It was my bad, I was suppose to add 'Wildfly client library' and 'JBoss EAP client library' was for the JBEAP issue.
"Jakarta Enterprise Beans and Jakarta Messaging client library"? Would be good to add.
Thanks.